### PR TITLE
Deprecate v1alpha1 CRDs

### PR DESCRIPTION
**Pull request by TheSpiritXIII**

### CODE REVIEW

File: cmd/operator/deploy/crds/monitoring.googleapis.com_clusterpodmonitorings.yaml [hunk 1]
```diff
@@ -293,6 +293,8 @@ spec:
     subresources:
       status: {}
   - name: v1alpha1
+    deprecated: true
+    deprecationWarning: Use version v1
```
Comment (liggitt) [inline cmd/operator/deploy/crds/monitoring.googleapis.com_clusterpodmonitorings.yaml:297]: I'd recommend omitting `deprecationWarning` and using the standard warning message that recommends v1

Comment (TheSpiritXIII) [reply cmd/operator/deploy/crds/monitoring.googleapis.com_clusterpodmonitorings.yaml]: Great suggestion! Done.

**Comment by liggitt** [comment]
lgtm

**Review by pintohutch — COMMENTED** [review]
You'll probably want to add `OperatorConfig` to the deprecated APIs.

**Review by pintohutch — APPROVED** [review]

**Pull request merged**